.bazelrc

```diff
@@ -15,7 +15,7 @@ build --ios_simulator_version=15.0
 build --spawn_strategy=local
 build --verbose_failures
 build --workspace_status_command=envoy/bazel/get_workspace_status
-build --xcode_version=13.0
+build --xcode_version=13.1
```

Comment:
Do you also need to update `ci/mac_ci_setup.sh`? https://github.com/envoyproxy/envoy-mobile/blob/e41c2b65c7d1c523b99675174a60c8f77393841a/ci/mac_ci_setup.sh#L57

Reply:
I see that wasn't sufficient. Is this because the compilation actually happens on remote execution on EngFlow and we need to add Xcode 13.1-enabled workers there?

Reply:
yep!